justifi-tech/web-component-library
# add spinner and missing var

QA plan:
- Submit a payment on a hosted checkout session. (Sorry you have to run it locally Kristi 😞 )

Expect:
- Payment form works the same as it was, except there's a spinner while submitting.
- When the Payment form is styled, the loading state is styled properly and not change unexpectedly.

Changes:
- Add separate loading state from enabled state.
- Add missing variables for loading state.
- Add spinner when loading.

<img width="1104" alt="image" src="https://github.com/justifi-tech/web-component-library/assets/10523683/c792de8e-fefc-476b-bc93-8cd30d1189b0">
<img width="1105" alt="image" src="https://github.com/justifi-tech/web-component-library/assets/10523683/302aa6ac-4a60-4aff-94c7-390443133934">
